python/feldera/enums.py

```diff
@@ -7,27 +7,169 @@ class CompilationProfile(Enum):
     """
 
     SERVER_DEFAULT = None
+    "The compiler server default compilation profile."
```

Comment:
Should this not be docstring, with `"""..."""`? I think it is currently just an unused string statement.